README.md

```diff
@@ -17,44 +17,60 @@ NOTE: We do not guarantee the quality of responses produced by this sample copil
 
 ## Step 1: Set up your development environment
 
-#### Use a pre-built development environment
-To get started quickly, you can use a pre-built development environment. **Click the button below** to open the repo in GitHub Codespaces, and then continue the readme!
-
+### Step 1a: Use a cloud development environment
+#### Explore sample with Codespaces
+- To get started quickly with this sample, you can use a pre-built Codespaces development environment. **Click the button below** to open this repo in GitHub Codespaces, and then continue the readme!
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/Azure/aistudio-copilot-sample?quickstart=1)
 
-Once you've opened in Codespaces you can proceed to the next step.
+- Once you've launched Codespaces you can proceed to step 2.
+
+#### Start developing in an Azure AI curated VS Code development environment
```

Comment:
Maybe we can use consistent word for both this item and "Explore sample with Codespaces"? -- e.g. we can change this one to "Explore sample with Azure AI curated VS Code development environment.

Reply:
This is intentional, because Codespaces and VS Code environment have different use cases.
If you are just exploring the sample, Codespaces may be quicker and less overhead
but If you plan to write your own code, we recommend the VS Code environment.